src/_locales/zh/messages.json

```diff
@@ -159,6 +159,8 @@
     "post_dialog__select_recipients_share_to_everyone": "任何人",
     "post_dialog__select_recipients_title": "選擇收件人",
     "post_dialog__select_specific_friends_title": "選擇好友 (已選 {{selected}})",
+    "post_dialog__more_options_title": "更多選項",
+    "post_dialog__image_payload": "圖片編碼",
```

Comment:
"隱寫到圖片"

**隐写** 同时满足 中国大陆地区/台湾地区/香港地区 对于概念的基本认知

Reference:
https://www.grb.gov.tw/search;keyword=%E9%9A%B1%E5%AF%AB;type=GRB05;scope=1
https://www.police.gov.hk/offbeat/996/chi/491.html

Reply:
In a previous discussion we decided to avoid using "Steganography", because it is a terminology in cryptography that does not refer to practices which do not try to conceal the fact the certain information is being embedded in the image while the current implementation of "Image-Based Payload" does not try so.

Reply:
"隱藏到圖片"